api/serverpb/server.proto

```diff
@@ -147,6 +149,9 @@ message ChangeSettingsRequest {
   bool enable_stt = 11;
   // Disable Security Threat Tool
   bool disable_stt = 12;
+  // PMM Server public address.
+  string pmm_public_address = 13;
```

Comment:
What about pmm external address? :)

Reply:
isn't it the same? public sounds better for me :)

Reply:
This is not critical. :)
The word "public" has the connotation "public place". pmm-server not necessary to be publicly available for everybody.  It only must be available for external to its services.